source/form.html

```diff
@@ -50,40 +50,155 @@ <h2 class="page__heading">Оставьте свой отзыв
+          <li class="checkbox-field">
+            <label class="checkbox-field__label" for="place2-field">Гора-Колокол
+            </label>
+            <input class="checkbox-field__input" type="checkbox" name="place2" id="place2-field" checked>
+          </li>
+          <li class="checkbox-field">
+            <label class="checkbox-field__label" for="place3-field">Слайд-парк
+            </label>
+            <input class="checkbox-field__input" type="checkbox" name="place3" id="place3-field" checked>
+          </li>
+          <li class="checkbox-field">
+            <label class="checkbox-field__label" for="place4-field">Красные скалы
+            </label>
+            <input class="checkbox-field__input" type="checkbox" name="place4" id="place4-field" checked>
+          </li>
+        </ul>
+      </fieldset>
+      <div class="feedback-form__group comment-group">
+       <h3 class="feedback-form__heading">Опишите свои эмоции:
+       </h3>
+       <textarea class="comment-group__textarea" name="comment" id="comment-field" rows="6">
+         Опишите подробно все свои восторги
+       </textarea>
+     </div>
+     <button class="feedback-form__button button" type="submit">Отправить отзыв
+     </button>
+   </form>
+   <div class="popup popup--failure">
+    <h2 class="popup__heading">Что-то пошло не так!
+    </h2>
+    <p class="popup__info">
+      Проверьте поля, выделенные красным, скорее всего Вы забыли их заполнить
     </p>
-    <section class="feedback">
-     <form class="feedback__form form" action="https://echo.htmlacademy.ru" method="post">
-     </form>
-    </section>
-  </main>
-  <footer class="footer">
-    <b class="footer__hashtag">#WISITSEDONA</b>
-    <ul class="footer__list social-btn-list">
-      <li class="social-btn-list__item">
-        <a class="social-btn-list__link" href="https://twitter.com/htmlacademy_ru">
-          <img class="social-btn-list__icon" src="https://via.placeholder.com/47x47/88cc00/"
-          alt="Логотип Твиттер">
-        </a>
-      </li>
-      <li class="social-btn-list__item">
-        <a class="social-btn-list__link" href="https://facebook.com/htmlacademy">
-          <img class="social-btn-list__icon" src="https://via.placeholder.com/47x47/88cc00"
-          alt="Логотип Фейсбук">
-        </a>
-      </li>
-      <li class="social-btn-list__item">
-        <a class="social-btn-list__link" href="https://youtube.com/htmlacademyru">
-          <img class="social-btn-list__icon" src="https://via.placeholder.com/47x47/88cc00"
-          alt="Логотип Ютуб">
-        </a>
-      </li>
-    </ul>
-    <div class="footer__copyright copyright">
-      <span class="copyright__text">Разработано:</span>
-      <a class="copyright__link" href="https://htmlacademy.ru/intensive/adaptive">
-        <img class="copyright__img" src="https://via.placeholder.com/28x36/88cc00"
-        alt="Логотип HTML академии">
+    <button class="popup__button button" type="submit">Ок</button>
```

Comment:
```suggestion
    <button class="popup__button button" type="button">Ок</button>
```

Здесь нет формы, а `type="submit"` для отправки данных на сервер.